Aztec/Classes/TextKit/ParagraphStyle.swift

```diff
@@ -220,6 +244,41 @@ open class ParagraphStyle: NSMutableParagraphStyle, CustomReflectable {
     }
 
     open override var description: String {
-        return super.description + " Blockquote: \(String(describing:blockquote)),\n HeaderLevel: \(headerLevel),\n HTMLParagraph: \(String(describing: htmlParagraph)),\n TextLists: \(textLists)"
+        return super.description + " Blockquotes: \(String(describing:blockquotes)),\n HeaderLevel: \(headerLevel),\n HTMLParagraph: \(String(describing: htmlParagraph)),\n TextLists: \(textLists)"
+    }
+}
+
+// MARK: - Add method to manipulate properties array
+
+extension ParagraphStyle {
+
+    func add(property: ParagraphProperty) {
+        properties.append(property)
+    }
+
+    func removeProperty(ofType type: AnyClass) {
+        var position: Int?
+        for index in (0..<properties.count).reversed() {
+            if type(of: properties[index]) == type {
+                position = index
```

Comment:
Any reason not to remove the property right here?  It would make for shorter / cleaner code, and since we're breaking out of the loop immediately it shouldn't break the enumerator.